lang/java/reef-examples/src/main/java/org/apache/reef/examples/suspend/SuspendTestTask.java

```diff
@@ -169,13 +169,11 @@ private synchronized void restore(final byte[] memento) throws IOException, Inte
    * Handler for suspend event.
    */
   public class SuspendHandler implements EventHandler<SuspendEvent> {
-
     @Override
     public void onNext(final SuspendEvent suspendEvent) {
-      final byte[] message = suspendEvent.get().get();
       synchronized (SuspendTestTask.this) {
-        LOG.log(Level.INFO, "Suspend: {0} with: {1} bytes; counter: {2}",
-            new Object[]{this, message.length, SuspendTestTask.this.counter});
```

Comment:
Actually, I don't know the functionality covered by TestSuspendTask.  And I don't change functionality in this example. In this example, the driver doesn't send a message to the task when the driver let the task suspend.
So I delete ``final byte message = suspendEvent.get().get()``, because the message.length makes null pointer exception.